manifests/k/kelteseth/ScreenPlay/Beta/0.15.1-beta1/kelteseth.ScreenPlay.Beta.installer.yaml

```diff
@@ -0,0 +1,23 @@
+# Created with komac v2.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: kelteseth.ScreenPlay.Beta
+PackageVersion: 0.15.1-beta1
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: bin/ScreenPlay.exe
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: Microsoft.VCRedist.2015+.x64

```

Comment:
Should `Valve.Steam` be a dependency? If not, it's okay, as this message makes it obvious without blocking or erroring:
![image](https://github.com/microsoft/winget-pkgs/assets/114952565/77fda6d5-3634-4c3e-929d-add33e1d1681)

```suggestion
  - PackageIdentifier: Microsoft.VCRedist.2015+.x64
  - PackageIdentifier: Valve.Steam
```